### PR TITLE
feat: explain why AWS Config Service is important

### PR DESCRIPTION
**Pull request by manuelrodriguez-nr**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx [hunk 1]
```diff
@@ -319,11 +322,11 @@ The CloudWatch metric stream integration automatically ingests new metrics confi
 
 ### Custom metrics [#custom-metrics]
 
-To ingest CloudWatch custom metrics, make sure your custom namespace is visible in the CloudWatch metric stream configuration and it's not being filtered by inclusion or exclusion rules.  
+To ingest CloudWatch custom metrics, make sure your custom namespace is visible in the CloudWatch metric stream configuration and it's not being filtered by inclusion or exclusion rules.
 
 ### Percentiles [#percentiles]
 
-AWS CloudWatch allows you to [define additional statistics](https://aws.amazon.com/about-aws/whats-new/2022/04/amazon-cloudwatch-metric-streams-additional-statistics/), including percentiles. 
+AWS CloudWatch allows you to [define additional statistics](https://aws.amazon.com/about-aws/whats-new/2022/04/amazon-cloudwatch-metric-streams-additional-statistics/), including percentiles.
```
Comment (manuelrodriguez-nr) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx:329]: lint, can be ignored

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx [hunk 2]
```diff
@@ -63,7 +63,7 @@ This table outlines the various parameters required for the CloudFormation templ
       <th>
         Description
       </th>
-      
+
```
Comment (manuelrodriguez-nr) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx:66]: lint, can be ignored

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 3]
```diff
@@ -75,9 +75,11 @@ Consider the following when evaluating the cost of the AWS CloudWatch metric str
 * [AWS Kinesis Firehose ingest](https://aws.amazon.com/kinesis/data-firehose/pricing/).
 * [AWS Kinesis Firehose data transfer](https://aws.amazon.com/kinesis/data-firehose/pricing/).
 * Optional: [AWS Config service](https://aws.amazon.com/config/pricing/) used to enrich metrics with resource metadata on select AWS namespaces.
-
+<Callout variant="tip">
+New Relic needs access to AWS Config Service to detect, identify and monitor your AWS services. Disabling the access will lead to an incomplete monitoring and poor New Relic experience.
+</Callout>
 Learn about the mechanisms available for [data management](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup#manage-data), including filters in AWS and New Relic side.
-When applicable, make sure to complete an initial integration in a pre-production environment to evaluate the total cost of the solution based on a limited number of AWS resources and services. 
+When applicable, make sure to complete an initial integration in a pre-production environment to evaluate the total cost of the solution based on a limited number of AWS resources and services.
```
Comment (manuelrodriguez-nr) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:82]: lint, can be ignored

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 4]
```diff
@@ -114,7 +116,7 @@ Please consider the following when migrating from poll-based integrations:
 * **Dashboards**: Custom dashboards that use poll-based AWS integration events will still work as expected.
 * **Alerts**: Alert conditions that use poll-based AWS events will still work. We recommend adapting those to the dimensional metric format (using NRQL as source).
 * **Entities**: New Relic Explorer might show duplicated entities for up to 24 hours.
-* **Attributes**: poll-based AWS integrations prefix collected resource tags with `label.`, while the AWS CloudWatch Metric Streams integration prefixes collected resource tags with `tags.`. If both integrations are enabled for the same AWS account, resource tags will appear under both prefixes when using the Event format. 
+* **Attributes**: poll-based AWS integrations prefix collected resource tags with `label.`, while the AWS CloudWatch Metric Streams integration prefixes collected resource tags with `tags.`. If both integrations are enabled for the same AWS account, resource tags will appear under both prefixes when using the Event format.
```
Comment (manuelrodriguez-nr) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:119]: lint, can be ignored

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 5]
```diff
@@ -135,5 +137,5 @@ Integrations based on CloudWatch Logs, [forwarded to New Relic via Lambda](/docs
 
 ## Metrics not available with CloudWatch metric streams
 
-AWS CloudWatch metric stream will not include metrics that are made available to CloudWatch with more than 2 hours delay. Examples of AWS namespaces that might contain metrics that are aggregated and exposed after 2 hours include: AWS DMS, AWS RDS, AWS DocDB, AWS S3 and AWS DAX.  
+AWS CloudWatch metric stream will not include metrics that are made available to CloudWatch with more than 2 hours delay. Examples of AWS namespaces that might contain metrics that are aggregated and exposed after 2 hours include: AWS DMS, AWS RDS, AWS DocDB, AWS S3 and AWS DAX.
```
Comment (manuelrodriguez-nr) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:140]: lint, can be ignored

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx [hunk 6]
```diff
@@ -290,8 +290,11 @@ Note that not all metrics have their custom tags as dimensions. Currently, only
 
 ## Metadata collection [#metadata-collection]
 
-Like with custom tags, New Relic also pulls metadata information from relevant AWS services in order to decorate AWS CloudWatch metrics with enriched metadata collected from AWS Services APIs. This metadata is accessible in New Relic as additional dimensions on the metrics provided by AWS CloudWatch. This is an optional capability that's complementary to the CloudWatch Metric Streams integration.
+Like with custom tags, New Relic also pulls metadata information from relevant AWS services in order to decorate AWS CloudWatch metrics with enriched metadata collected from AWS Services APIs. This metadata is accessible in New Relic as additional dimensions on the metrics provided by AWS CloudWatch.
 
+  <Callout variant="important">
+This is an optional capability that's complementary to the CloudWatch Metric Streams integration. However, it is necessary to have a full New Relic experience and not enabling it will lead to incomplete service discovery and monitorization.
+  </Callout>
```
Comment (manuelrodriguez-nr) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx:297]: this is an important change

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx [hunk 7]
```diff
@@ -319,11 +322,11 @@ The CloudWatch metric stream integration automatically ingests new metrics confi
 
 ### Custom metrics [#custom-metrics]
 
-To ingest CloudWatch custom metrics, make sure your custom namespace is visible in the CloudWatch metric stream configuration and it's not being filtered by inclusion or exclusion rules.  
+To ingest CloudWatch custom metrics, make sure your custom namespace is visible in the CloudWatch metric stream configuration and it's not being filtered by inclusion or exclusion rules.
```
Comment (manuelrodriguez-nr) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx:325]: lint, can be ignored

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 8]
```diff
@@ -75,9 +75,11 @@ Consider the following when evaluating the cost of the AWS CloudWatch metric str
 * [AWS Kinesis Firehose ingest](https://aws.amazon.com/kinesis/data-firehose/pricing/).
 * [AWS Kinesis Firehose data transfer](https://aws.amazon.com/kinesis/data-firehose/pricing/).
 * Optional: [AWS Config service](https://aws.amazon.com/config/pricing/) used to enrich metrics with resource metadata on select AWS namespaces.
-
+<Callout variant="tip">
+New Relic needs access to AWS Config Service to detect, identify and monitor your AWS services. Disabling the access will lead to an incomplete monitoring and poor New Relic experience.
+</Callout>
```
Comment (manuelrodriguez-nr) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:80]: this is an important addition

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx (hunk 6 above)
Comment (manuelrodriguez-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx:297]: my question here is, how deeply should we explain this? Should we go into details or this generic warning is enough?

Comment (manuelrodriguez-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx:297]: Screenshot 2023-01-30 at 09 17 42

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx (hunk 8 above)
Comment (manuelrodriguez-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:80]: Screenshot 2023-01-30 at 09 18 54

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx [hunk 9]
```diff
@@ -147,7 +147,7 @@ This table outlines the various parameters required for the CloudFormation templ
       </td>
 
       <td>
-        Enable and configure AWS Config to track resource changes
+        Enable and configure AWS Config to track resource changes. Disabling this will lead to an incomplete monitoring and poor New Relic experience.
```
[the remark this answers is not in the document]
Comment (manuelrodriguez-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx:150]: Screenshot 2023-01-30 at 09 15 38

**Review by naxhh — APPROVED** [review]

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 10]
```diff
@@ -75,9 +75,11 @@ Consider the following when evaluating the cost of the AWS CloudWatch metric str
 * [AWS Kinesis Firehose ingest](https://aws.amazon.com/kinesis/data-firehose/pricing/).
 * [AWS Kinesis Firehose data transfer](https://aws.amazon.com/kinesis/data-firehose/pricing/).
 * Optional: [AWS Config service](https://aws.amazon.com/config/pricing/) used to enrich metrics with resource metadata on select AWS namespaces.
-
```
Comment (manuelrodriguez-nr) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:78]: ALEIX: https://docswebsitedevelop-manuelrodrigueznrdocswebs64650.gatsbyjs.io/docs/infrastructure/amazon-integrations/connect/aws-metric-stream/

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx [hunk 11]
```diff
@@ -63,7 +63,7 @@ This table outlines the various parameters required for the CloudFormation templ
       <th>
         Description
       </th>
-
```
Comment (manuelrodriguez-nr) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx:66]: ALEIX: https://docswebsitedevelop-manuelrodrigueznrdocswebs64650.gatsbyjs.io/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup/

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx (hunk 8 above)
Comment (manuelrodriguez-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx]: ^^ the screenshoot is outdated, please check the source code directly

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx (hunk 6 above)
Comment (manuelrodriguez-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx:297]: ^^ the screenshoot is outdated, please check the source code directly

**Review by nbaenam — DISMISSED** [review]

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx [hunk 12]
```diff
@@ -39,17 +40,17 @@ Before CloudWatch metric streams, the only solution for AWS monitoring partners
       <td>
         **All CloudWatch metrics from all AWS services and custom namespaces are available** in New Relic at once, without needing a specific integration to be built or updated.
 
-        There's one exception: metrics that are made available to CloudWatch with more than 2 hours delay are not included in the stream.
+        There's one exception: When metrics are available to AWS CloudWatch metric stream with more than 2 hours delay, these metrics aren't included in the stream.
```
Comment (mkrichevsky) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream.mdx:43]: metrics are made available in `CloudWatch` service, not metric stream.
Probably correct way would be `...When metrics are available to AWS CloudWatch with more than...`

File: src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx [hunk 13]
```diff
@@ -290,7 +291,11 @@ Note that not all metrics have their custom tags as dimensions. Currently, only
 
 ## Metadata collection [#metadata-collection]
 
-Like with custom tags, New Relic also pulls metadata information from relevant AWS services in order to decorate AWS CloudWatch metrics with enriched metadata collected from AWS Services APIs. This metadata is accessible in New Relic as additional dimensions on the metrics provided by AWS CloudWatch. This is an optional capability that's complementary to the CloudWatch Metric Streams integration.
+Like with custom tags, New Relic also pulls metadata information from relevant AWS services in order to decorate AWS CloudWatch metrics with enriched metadata collected from AWS Services APIs. This metadata is accessible in New Relic as additional dimensions on the metrics provided by AWS CloudWatch.
+
+  <Callout variant="important">
+This optional capability is complementary to the CloudWatch Metric Streams integration. Your service discovery and monitoring will be incomplete if you don't enable this optional capability.
```
Comment (mkrichevsky) [inline src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx:297]: do we want to mention any specific capabilities that won't work if Config is not enabled?

Comment (manuelrodriguez-nr) [reply src/content/docs/infrastructure/amazon-integrations/connect/aws-metric-stream-setup.mdx:297]: I think that would be a bit too low level, and also we would need to keep an eye on it so it doesn't get updated. But if you think it is worth it, we can of course specify it

**Review by mkrichevsky — APPROVED** [review]

**Review by nbaenam — APPROVED** [review]

**Pull request merged**